rosdep/base.yaml

```diff
@@ -2612,6 +2612,23 @@ mpg123:
   debian: [mpg123]
   fedora: [mpg123-devel]
   ubuntu: [mpg123]
+mplayer:
+  arch: [mplayer]
+  debian:
+    jessie: [mplayer]
+    sid: [mplayer]
+    squeeze: [mplayer]
+    wheezy: [mplayer]
+  fedora: [mplayer]
+  ubuntu:
+    precise: [mplayer]
```

Comment:
Since these entries are all the same please put a single entry under ubuntu and debian like you did with fedora instead of calling out each individual version.